### ISSUE TITLE
`sz lh add` should have the option to automatically add a commit

### ISSUE THREAD

**Issue by Jonas-Sander** [SharezoneApp/sharezone-app#1549]
When running `sz lh add` one should have the option that the command is run and a commit with the message: "Add license headers" is automatically created and comitted.

We might even do it automatically e.g. if git has no staged/changed(?) files when running this command.

**Comment by Jonas-Sander** [comment]
> We might even do it automatically e.g. if git has no staged/changed(?) files when running this command.

@nilsreichardt WDYT?

**Comment by nilsreichardt** [comment]
Do you mean the command `sz lh add` is executed automatically (when exactly is it executed) or when running `sz lh add` runs it's automatically commit?

**Comment by Jonas-Sander** [comment]
The latter.

**Comment by nilsreichardt** [comment]
Yes, we can do that when there are no other changed files before running the command. We could add it as a flag which is enabled by default
